presto-docs/src/main/sphinx/release/release-0.289.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.289
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in the hash code of -0 for double and real types when ``use-new-nan-definition`` is false :pr:`23060`.
```

Comment:
```suggestion
* Fix a bug in the hash code of -0 for double and real types when ``use-new-nan-definition`` is false. :pr:`23060`
```
Nit of punctuation, moving the ``.`` from after the PR # to the end of the sentence. Edit applied throughout for consistency with the [0.288](https://prestodb.io/docs/current/release/release-0.288.html), [0.287](https://prestodb.io/docs/current/release/release-0.287.html), and [0.286](https://prestodb.io/docs/current/release/release-0.286.html) release notes.